controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -109,3 +472,17 @@ func (r *StorageClassClaimReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&ocsv1alpha1.StorageClassClaim{}).
 		Complete(r)
 }
+
+func (r *StorageClassClaimReconciler) get(obj client.Object) error {
+	key := client.ObjectKeyFromObject(obj)
+	return r.Client.Get(r.ctx, key, obj)
+}
```

Comment:
I see no value add for this utility. What am I missing?

Reply:
Resolving this as we agreed to settle the difference in approaches as tech debt